robottelo/common/helpers.py

```diff
@@ -151,6 +151,32 @@ def generate_strings_list(len1=None):
     ]
 
 
+def generate_strings_dict(len1=None):
```

Comment:
OR may be I will see whether it's possible to merge the below methods to one single method.
1) generate_strings_list()
2) removekeys
3) generate_strings_dict.

I will give a try.